### PR TITLE
Added DynamoDb Persistence Adapter implementation.

### PR DESCRIPTION
**Pull request by ncipollina**

### CODE REVIEW

File: src/Ask.Sdk.DynamoDb.Persistence.Adapter/Attributes/Persistence/DynamoDbPersistenceAdapter.cs [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                    {
+                        new KeySchemaElement
+                        {
+                            AttributeName = partitionKeyName,
+                            KeyType = KeyType.HASH
+                        }
+                    },
+                    ProvisionedThroughput = new ProvisionedThroughput
+                    {
+                        ReadCapacityUnits = 5,
```
Comment (zakkl13) [inline src/Ask.Sdk.DynamoDb.Persistence.Adapter/Attributes/Persistence/DynamoDbPersistenceAdapter.cs]: Should these be optionally configurable through the Init method?

File: samples/DynamoDbLambda/Readme.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# AWS Lambda Empty Function Project
```
Comment (zakkl13) [inline samples/DynamoDbLambda/Readme.md]: Can you update the readme to provide a brief description of what this sample demonstrates?

File: src/Ask.Sdk.DynamoDb.Persistence.Adapter/Attributes/Persistence/DynamoDbPersistenceAdapter.cs (hunk 1 above)
Comment (ncipollina) [reply src/Ask.Sdk.DynamoDb.Persistence.Adapter/Attributes/Persistence/DynamoDbPersistenceAdapter.cs]: @zakkl13, potentially, but I was just following with the NodeJS implementation.

File: samples/DynamoDbLambda/Readme.md (hunk 2 above)
Comment (ncipollina) [reply samples/DynamoDbLambda/Readme.md]: I can do that.

**Pull request merged**